app/src/main/java/ar/scacchipa/twittercloneapp/component/MainCloneActivity.kt

```diff
@@ -0,0 +1,60 @@
+package ar.scacchipa.twittercloneapp.component
+
+import android.os.Bundle
+import android.view.View
+import androidx.appcompat.app.ActionBar
+import androidx.appcompat.app.AppCompatActivity
+import androidx.core.content.ContextCompat
+import androidx.navigation.fragment.NavHostFragment
+import androidx.navigation.ui.AppBarConfiguration
+import androidx.navigation.ui.setupActionBarWithNavController
+import androidx.navigation.ui.setupWithNavController
+import ar.scacchipa.twittercloneapp.R
+import ar.scacchipa.twittercloneapp.databinding.ActivityMainLayoutBinding
+import com.google.android.material.bottomnavigation.BottomNavigationView
+
+class MainCloneActivity: AppCompatActivity() {
+
+    private lateinit var binding: ActivityMainLayoutBinding
+
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+
+        binding = ActivityMainLayoutBinding.inflate(layoutInflater)
+        setContentView( binding.root )
+
+        binding.mainBottomNavigationView.itemIconTintList = null
+        binding.mainBottomNavigationView.itemActiveIndicatorColor = null
+
+        val bottomNavView: BottomNavigationView = binding.mainBottomNavigationView
+
+        val navHostFragment = supportFragmentManager
+            .findFragmentById(R.id.main_nav_host_fragment) as NavHostFragment
+        val navController = navHostFragment.navController
+        navController.setGraph(R.navigation.main_nav_graph)
+
+        val appBarConfiguration = AppBarConfiguration(
+            setOf(
+                R.id.fragment_home_nav,
+                R.id.fragment_search_nav,
+                R.id.fragment_bell_nav,
+                R.id.fragment_message_nav
+            )
+        )
+
+        setupActionBarWithNavController(navController, appBarConfiguration)
+        bottomNavView.setupWithNavController(navController)
+
+        this.window?.apply {
```

Comment:
I think we can investigate if we can do this via a theme.

Reply:
It was moved to Theme.xml

        <item name="android:background">@color/white</item>
        <item name="android:statusBarColor">@color/white</item>
        <item name="android:navigationBarColor">@color/on_primary</item>
        <item name="android:windowLightStatusBar">true</item>

I was not able to move the supportActionBar to theme.xml. I found the following code but doesn't  works.

        <item name="android:elevation">0dp</item>
        <item name="android:displayOptions">showCustom</item>

or
         <item name="elevation">0dp</item>
        <item name="displayOptions">showCustom</item>